src/configure/configure.ts

```diff
@@ -162,13 +162,14 @@ class Orchestrator {
 
         if (this.continueOrchestration) {
             await this.getSourceRepositoryDetails();
-            await this.getAzureSession();
+            this.inputs.azureSession = getAzureSession();
```

Comment:
for web app right click scenario, we should pass the subscriptionId as it is already known and we will not be explicitly setting subscription later.

Reply:
added condition